search.nim

```diff
@@ -403,6 +404,9 @@ proc replaceAll*(win: var utils.MainWin, find, replace: cstring): int =
       found = ret[2]
   
     if found:
+      if addr(startMatch).getOffset() <= lastPos:
+        break
+      lastPos = addr(startMatch).getOffset()
```

Comment:
They're all replaced, I don't believe any of 'Replace All' depends on cursor position.